app/helpers/procedure_helper.rb

```diff
@@ -70,4 +70,11 @@ def admin_procedures_back_path(procedure)
     end
     admin_procedures_path(statut:)
   end
+
+  def can_recreate_a_dossier_from_a_procedure?(procedure)
+    procedure.closing_reason_internal_procedure? &&
+    procedure.replaced_by_procedure.present? &&
+    !procedure.replaced_by_procedure.discarded? &&
+    procedure.replaced_by_procedure.path.present?
```

Comment:
je suppose que la ligne 78 est juste la le temps que la data soit nettoyée. peut être rajouter un `TODO: to remove when all path are added` histoire que ce soit un no brainer pour le nous de demain ?